nl_server/gcs.py

```diff
@@ -69,7 +71,7 @@ def download_model_from_gcs(gcs_bucket: Any, local_dir: str,
 # Downloads the `model_folder` from GCS to /tmp/
 # and return its path.
 def download_model_folder(model_folder: str) -> str:
-  sc = storage.Client()
+  sc = storage.Client.create_anonymous_client()
```

Comment:
Comment why so?  Also this gets called from multiple places

Reply:
Done.